front/components/menu/menu_directives.js

```diff
@@ -1,4 +1,4 @@
-angular.module('necs.MenuModule.directives', ['ui.bootstrap'])
+angular.module('vc.MenuModule.directives', ['ui.bootstrap'])
 .directive('menuDirective',
 [
 function(cakeBaseUrl){
```

Comment:
fix spacing in controller function and make sure the comments are there.  It is good to be consistent.

Also, if you are going to have $scope.isStatus, and there is nothing else that is going to be used with it expect 'isopen', then just change it to 'isopen', but isopen is not correctly camelcased, so change that to 'isOpen'.

Also, remove the $log

Lastly, I question the need to preventDefault and stopPropagation.  That really shouldn't be needed...